tests/functional/test_zilien_reader.py

```diff
@@ -168,3 +225,71 @@ def test_read(cls_or_technique, expected_series):
         assert time_data[-1] == approx(single_series_details.t_last)
         assert value_data[0] == approx(single_series_details.v_first)
         assert value_data[-1] == approx(single_series_details.v_last)
+
+
+class TestZilienIntegrated:
+    """Tests for reading the Zilien files with integrated Biologic dataset."""
+
+    @pytest.mark.external
+    def test_series_match(self, datasets):
+        """Test presence of all the series from the mpt files in the tsv file."""
+
+        # coming from parametrized fixture
+        tsv, mpts, _ = datasets
+
+        # if one name from a mpt file would match a name from a different mpt
+        # in the integrated tsv dataset, then the test for series amount will
+        # fail, so it is not necessary to check for the exact belonging
+        for mpt in mpts:
+            for mpt_name in mpt.series_names:
+                if mpt_name == "time/s":
+                    assert f"Biologic {mpt_name}" in tsv.series_names
```

Comment:
I don't get this. How does this end up in `series_names`? Why put it there?

Reply:
The `Biologic time/s` is a column with timestamps from the Biologic measurement. So it needs to be created as a `TimeSeries` that is used as a `tseries` attribute for the following `ValueSeries`.